Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
-edition = "2021"
 name = "nftables"
 version = "0.5.0"
-license = "MIT OR Apache-2.0"
+authors = ["Jasper Wiegratz <wiegratz@uni-bremen.de>"]
```

Comment:
The adjusted `authors` field could look like this :)

```suggestion
authors = ["Jasper Wiegratz <wiegratz@uni-bremen.de>", "Jan Romann <jan.romann@uni-bremen.de>"]
```

Reply:
thx, done

Reply:
Thank you :)